### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by BastianZim**
Hi, the pyproject.toml lists MIT so I added it but not sure if this is correct.

I would just need the actual file for downstream (conda-forge).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 PyScript
```
Comment (BastianZim) [inline LICENSE:3]: Let me know if someone else has the Copyright

Comment (BastianZim) [reply LICENSE]: Had a sec and updated it but feel free to also close it.

Comment (mattkram) [reply LICENSE]: Thanks for the prompt. This is obviously a very new project.

I really appreciate the help getting it onto conda-forge!

Comment (BastianZim) [reply LICENSE]: Yes of course totally understood. Just wanted to get this into conda-forge so that I can try it :)

**Pull request closed without merging**